cmd/proxy-to-grafana/proxy-to-grafana.go

```diff
@@ -0,0 +1,121 @@
+// proxy-to-grafana uses a reverse proxy which identifies users based on their originating tailscale node and creates / logs in users to grafana. Uses the auth proxy feature for grafana: https://grafana.com/docs/grafana/latest/auth/auth-proxy/
+//
+// Provide the env var TS_AUTHKEY to have this server automatically join your Tailnet.
+//
+// Use this grafana configuration to enable the auth proxy:
+
```

Comment:
Need // here otherwise the packagedoc is only the comment adjacent to the package line